### DISCUSSION TITLE
Upload proof form broken?

### DISCUSSION THREAD

**Discussion by sarahciston** [community/community#116816] [GitHub Education]
### Hi everyone,

Hi! I received the notice about reverifying, and I have tried in both Firefox and Safari to upload documents with dates of my current university registration. I waited a few days and tried again. Each time the form appears to refresh and there is no notification that anything was successfully submitted. Today I got another reminder that my account will revert if I don't do anything. Is the form working? Was the proof rejected for some reason? Thanks for your help!

**Comment by JeremyKahn** [comment]
I have no answer but I had the same experience.

**Comment by zg04ckpt** [comment]
I was same you, but I fixed by adjust size of image (< 1MB)

**Reply by sarahciston** [reply]
Thanks, I double checked following your advice and, like the person below, I am still having the issue after ensuring my image size and resolution are within the correct boundaries. Thank you for flagging this, too.

**Comment by yogabonito** [comment]
Same (frustrating) experience here: After clicking *Process my application* the page reloads, but my application does not seem to have been sent 😤️
(I *did* ensure that the image is between 100KB and 1MB and that the minimum resolution requirement is met.)

**Reply by nihilisticzebra** [reply]
I'm already part of the program but GitHub for some reason won't let me reverify, it only wants to use the front camera from my phone for some reason and the picture is barely legible
